package.json

```diff
@@ -71,5 +71,8 @@
     "react-imageloader": "1.2.0",
     "react-intl": "1.2.0",
     "xhr": "2.0.1"
+  },
+  "jshintConfig": {
+    "esnext": true
```

Comment:
`src/lib/validators.js` uses [arrow functions](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions), which require the `esnext` option to be specified in our linter, or else it complains and bails.